mingw-w64-git/git-wrapper.c

```diff
@@ -64,8 +64,19 @@ static void find_exe_realpath(LPWSTR exepath, int count) {
 	if (dwlen >= MAX_PATH) {
 		wcscpy(exepath, mdexe);
 	} else {
-		size_t offset = wcsncmp(L"\\\\?\\", realexe, 4) ? 0 : 4;
-		size_t length = dwlen - offset;
+		size_t offset = 0;
+		size_t length = dwlen;
+
+		if (!wcsncmp(realexe, L"\\\\?\\", 4)) {
+			if (wcsncmp(realexe + 4, L"UNC\\", 4))
+				offset = 4;
+			else {
+				offset = 7;
```

Comment:
Why 7? what characters are we skipping over?

Reply:
`\\?\UNC`, and we will re-add a backslash so that the network path starts with a double backslash again.